src/devtools/client/debugger/src/reducers/pause.js

```diff
@@ -76,7 +76,7 @@ function update(state = createPauseState(), action) {
 
     case "FETCHED_FRAMES": {
       const { frames } = action;
-      const selectedFrameId = frames.length ? frames[0].id : undefined;
+      const selectedFrameId = frames && frames.length ? frames[0].id : undefined;
```

Comment:
perhaps we don't want to dispatch `FETCHED_FRAMES` if we don't have frames.

So basically put the guard in the action.

Reply:
we still need to set `framesLoading` to `false`, so we still need to dispatch `FETCHED_FRAMES`.

Reply:
good call.